docs/gestor_tareas.md

```diff
@@ -1,58 +1,14 @@
 # Opciones de gestor de tareas
 
-#### Mage
-**Mage**: Es una herramienta de automatización escrita en Go, que permite definir tareas directamente en este lenguaje. Estas tareas se configuran en el archivo `magefile.go` y se pueden ejecutar desde la línea de comandos.
-##### Ventajas:
+**Mage**: Al estar escrito en Go, Mage permite a los desarrolladores que ya conocen este lenguaje definir tareas sin tener que aprender nuevas sintaxis (como la de Make) o utilizar archivos YAML. Esto facilita la mantenibilidad y reduce la deuda técnica.
```

Comment:
No entiendo muy bien. ¿Cómo puede ser que de:

"Lo dice en el [guión](http://jj.github.io/IV/documentos/proyecto/3.Automatizar):
> En este caso habrá que establecer los criterios para elegir uno de ellos sobre otros y justificar la elección que eventualmente se tome.

Pero en todo caso debería estar claro que para documentar una decisión hay *siempre* que decir de antemano cuales son los criterios que se van a seguir.

"

Hayas entendido "voy a cambiar las cabeceras de orden 4 a netriga, explicar un poco más el Make y quitar las ventajas e inconvenientes de cada uno"?

Reply:
Como en el comentario anterior me dijiste: "No me tienes que contar una serie de herramientas elegidas aleatoriamente y contarme por qué te gusta o por qué no.", entendí que poner las ventajas y desventajas de cada uno no era incorrecto. 

Por otro lado, independientemente de lo que me comenta, ¿considera que la justificación que he completado sobre la elección en el último commit puede considerarse como válida?

Reply:
Recupero las ventajas e inconvenientes en: [3deed7a](https://github.com/antoniorr02/MenuConsulter/pull/15/commits/3deed7af4415ed1e310b8dcbe033639899592b9f)

Reply:
Te agradecería que leyeras **todo** el comentario, y por supuesto el guión.
Y te recuerdo lo que ya he repetido muchas veces. Si hacéis algo "porque el profe me lo ha dicho" es que no habéis entendido lo que ha dicho el profe.